docs/src/components/Footer.tsx

```diff
@@ -107,96 +98,72 @@ export default function Footer() {
               <StyledIconLink href="https://twitter.com/OrbitKiwi">
                 <Twitter />
               </StyledIconLink>
-            </Box>
-          </Box>
+            </Stack>
+          </Stack>
         </StyledInner>
       </StyledContainer>
       <StyledContainer color="paletteCloudNormal">
-        <StyledInner>
-          <Box
+        <StyledInner thick>
+          <Stack
+            direction="column"
             largeDesktop={{
-              display: "flex",
               align: "center",
               direction: "row",
               justify: "between",
             }}
           >
-            <Box tablet={{ display: "flex", align: "center" }}>
-              <Box
-                padding={{ top: "XSmall", bottom: "XSmall" }}
-                desktop={{ padding: { right: "XXLarge" } }}
-                largeDesktop={{ padding: { top: "large", bottom: "large" } }}
-              >
+            <Stack
+              inline
+              direction="column"
+              tablet={{ direction: "row", align: "center", spacing: "XXLarge" }}
+            >
+              <Stack inline grow={false} direction="column" spacing="none">
                 <Text as="div" weight="bold">
                   Design tokens
                 </Text>
                 <Text as="div" type="secondary">
                   v0.11.0
                 </Text>
-              </Box>
+              </Stack>
               <Dot />
-              <Box
-                padding={{ top: "XSmall", bottom: "XSmall" }}
-                tablet={{ padding: { top: "XSmall", bottom: "XSmall", left: "XXLarge" } }}
-                desktop={{
-                  padding: { right: "XXLarge", left: "XXLarge" },
-                }}
-                largeDesktop={{
-                  padding: { top: "large", right: "XXLarge", bottom: "large", left: "XXLarge" },
-                }}
-              >
+              <Stack inline grow={false} direction="column" spacing="none">
                 <Text as="div" weight="bold">
                   React components
                 </Text>
                 <Text as="div" type="secondary">
                   v0.11.0
                 </Text>
-              </Box>
+              </Stack>
               <Dot />
-              <Box
-                display="flex"
-                align="center"
-                padding={{ top: "XSmall", bottom: "XSmall" }}
-                tablet={{ padding: { top: "XSmall", bottom: "XSmall", left: "XXLarge" } }}
-                desktop={{ padding: { left: "XXLarge" } }}
-                largeDesktop={{ padding: { top: "large", bottom: "large", left: "XXLarge" } }}
-              >
-                <Box padding={{ right: "XLarge" }} desktop={{ shrink: 0 }}>
+              <Stack inline grow={false} spacing="none">
```

Comment:
You need to add spacing between label and the switch. This is how the footer looked at 367 px before:

![Screen Shot 2021-04-07 at 12 16 15](https://user-images.githubusercontent.com/471278/113854206-e8207400-979e-11eb-87cd-eb4a3dd0fcb5.png)

and after your change:

![Screen Shot 2021-04-07 at 12 15 47](https://user-images.githubusercontent.com/471278/113854183-df2fa280-979e-11eb-8f3f-173541c8d416.png)

Plus, the switch is overflowing the footer on the right (the screenshot is taken full-width).

You can also notice that the "Built & maintained by Kiwi.com" part is no longer centered, not sure if that was on purpose.